### PR TITLE
modified README to include link to final report

### PR DESCRIPTION
**Pull request by nphaterp**

### CODE REVIEW

File: README.Rmd [hunk 1]
```diff
@@ -33,6 +33,10 @@ and outcomes that these two countries have faced during the pandemic. However,
 further analysis is required to better understand the differences present in
 the Canada-US responses to COVID-19. 
 
+# Report
+
+The final report can be found [here](https://github.com/UBC-MDS/covid-19-cases-vs-tests-analysis/blob/main/doc/covid-response.pdf)
```
Comment (nichowu) [inline README.Rmd]: missing a 'period' at the end of the sentence.

File: README.Rmd [hunk 2]
```diff
@@ -33,6 +33,10 @@ and outcomes that these two countries have faced during the pandemic. However,
...
+
 # Usage
```
Comment (nichowu) [inline README.Rmd:40]: Just adding a to-do here: change the usage part to according to the Makefile.

Comment (nichowu) [reply README.Rmd:40]: another to-do: add "GNU make 4.2.1"  under dependencies.

Comment (nichowu) [inline README.Rmd:40]: Are we missing "os" and "calendar" packages that are needed in download.py and eda_codiv19.py?

**Review by nichowu — CHANGES_REQUESTED** [review]
requesting some changes.

**Comment by tanmaysharma19** [comment]
LGTM.

**Pull request merged**